README.md

```diff
@@ -29,26 +29,43 @@ Then, add the `dist` directory files into a relevant place in your code base —
 
 ```
 - uswds-0.9.0/
-  - dist/
-    - css/
-    - fonts/
-    - img/
-    - js/
+  ├── js
+  │   ├── uswds.min.js.map
+  │   ├── uswds.min.js
+  │   └── uswds.js
+  ├── css
+  │   ├── uswds.min.css.map
+  │   ├── uswds.min.css
+  │   └── uswds.css
+  ├── img/
+  │   ├── social-icons/
+  │   ├── favicons/
+  │   ├── alerts/
+  └── fonts/
 ```
 
+We offer two versions — a minified version, and an un-minified one. The minified
+versions contain a `.min` suffix in the filename before the file's extension.
+The un-minified versions do not contain the `.min` suffix.
+
+Use the minified version in a production environment or to reduce the file size
+of your downloaded assets. And the un-minified version is better if you are in a
+development environment or would like to debug the CSS or JavaScript assets in
+the browser. The examples below recommend using the minified versions.
+
 Refer to these files by adding the following `<link>` and `<script>` elements
 into your HTML pages:
 
 Add this to your `<head>` element:
 
 ```html
-<link rel="stylesheet" href="/path/to/your/assets/css/uswds-0.9.0.css">
+<link rel="stylesheet" href="/path/to/your/assets/css/lib/uswds.min.css">
 ```
 
 Add this before the closing `</body>` tag:
 
 ```html
-<script src="/path/to/your/assets/js/uswds-0.9.0.js"></script>
+<script src="/path/to/your/assets/js/lib/uswds.min.js"></script>
```

Comment:
I don't know if we want to include the folder. We name the folder `uswds-VERSION_NUMBER` so users can keep track of it when it gets unzipped, but we're assuming that they are copying the files out of that folder and putting it somewhere in their project right? not just sticking the whole folder into `/path/to/your/assets/**/*`